src/Resources/contao/languages/en/default.xlf

```diff
@@ -1679,6 +1679,9 @@ This e-mail has been generated by Contao. You can not reply to it directly.
       <trans-unit id="MSC.metaNoLanguages">
         <source>You have to add a root page in the site structure to store file meta information.</source>
       </trans-unit>
+      <trans-unit id="MSC.paginationAriaLabel">
+        <source>Pagination</source>
```

Comment:
If there are several navigation elements on one page, it is very helpful for screen reader users if you label them differently so that the user is able to distinguish their usage. Something like "main navigation", "breadcrumb navigation" etc.